src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Colors[] allColors;
+
+    public ColorSupplier() {
+        allColors = Colors.values();
+    }
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(allColors.length);
```

Comment:
Think about which variables should be local in the method and which should be class-level. new Random() object will be created on each getRandomBall() call. How can you avoid this?